util/perl/OpenSSL/config.pm

```diff
@@ -160,6 +160,11 @@ my $guess_patterns = [
     [ 'CYGWIN.*',                   '${MACHINE}-pc-cygwin' ],
     [ 'vxworks.*',                  '${MACHINE}-whatever-vxworks' ],
 
+    # Note: there's also NEO and NSR, but they are old and unsupported
+    [ 'NSE-.*?:NONSTOP_KERNEL:.*',  'nse-tandem-nsk${RELEASE}' ],
+    [ 'NSV-.*?:NONSTOP_KERNEL:.*',  'nsv-tandem-nsk${RELEASE}' ],
```

Comment:
-> `'nsv-tandem-nsk${RELEASE}'`

Reply:
This translates uname data to standard host triplets.  I'd rather keep those here.  However, the next place is where we convert the host triplet to OpenSSL config names, that's the proper spot for this kind of enforcement.